cmd/soroban-cli/src/commands/tx/update/sequence_number/next.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{
+    commands::{
+        global,
+        tx::xdr::{tx_envelope_from_input, Error as XdrParsingError},
+    },
+    config,
+    xdr::{self, SequenceNumber, TransactionEnvelope, WriteXdr},
+};
+
+#[derive(clap::Parser, Debug, Clone)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub config: config::Args,
+}
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    XdrStdin(#[from] XdrParsingError),
+    #[error(transparent)]
+    Xdr(#[from] xdr::Error),
+    #[error("only V1 transactions are supported")]
```

Comment:
I think it's more clear to write that v0 or fee bump transactions are not supported (as a user I know that there are regular transactions -- v1 and fee bump transactions, I assume v0 is legacy transactions)